scripts/build/gen_isr_tables.py

```diff
@@ -131,12 +131,51 @@ def parse_args():
 #endif
 """
 
+sh_isr_function = "__attribute__((section(\".shared_isr\"))) void shared_isr_"
+sh_isr_args = "const uint32_t shared_isr_"
```

Comment:
what about 64bit platforms?

Reply:
I've changed the uint32_t to size_t and tested it on arm32 and arm64. On arm64, I passed in a 64bit value as the argument to the ISR and verified there wasn't a build error and that the generated isr_tables.c had the correct value